setup.py

```diff
@@ -125,20 +125,20 @@ def get_es_requirements(es_version):
                   'cli',
                   'conf'],
         install_requires=[
-          'gevent >=1.0.1, <=1.2.2', # gevent version 1.0.0 do not support pyhton 2.7.8 https://github.com/gevent/gevent/issues/513
+          'gevent >=1.0.1, <1.3', # gevent version 1.0.0 do not support pyhton 2.7.8 https://github.com/gevent/gevent/issues/513
           # ES_VERSION env var is read in order to decide which version of the python library must be installed,
           # if ES_VERSION is no provided we assume that elasticsearch 2.x will be used.
           'elasticsearch {}'.format(get_es_requirements(os.environ.get('ES_VERSION', '2'))),
-          'flask-bootstrap <=4',
+          'flask-bootstrap <4',
           'Flask-Babel',
           'Flask-Authbone >=0.2.2',
-          'Flask <= 0.12',
+          'Flask <0.12',
```

Comment:
this would basically revert #327 (and is irrelevant to declared goal of PR)

Reply:
Ops... It should be `<0.13`, fixed with latest commit.